source/shared/cpp/ObjectModel/MarkDownBlockParser.h

```diff
@@ -97,6 +97,8 @@ namespace AdaptiveSharedNamespace
         // Callback function that handles the Emphasis State
         static EmphasisState MatchEmphasis(EmphasisParser&, std::stringstream&, std::string&);
 
+        static bool IsEmphasisToken(const int token);
```

Comment:
shouldn't need `const` here

Reply:
It isn't necessary but isn't it considered const-correct to have it when `token` should not be modified?

Reply:
current guidance is not to use `const` for simple params passed by value -- obviously that's not How I Was Raised, because I still `const` everything instinctively :)

Reply:
fixed in 4dd0da8